{
    "config": {
        "error": {
            "cannot_connect": "A liga\u00e7\u00e3o falhou",
            "timeout": "Excedido o tempo limite para estabelecer liga\u00e7\u00e3o",
            "unknown": "Erro inesperado"
        },
        "step": {
            "user": {
                "data": {
                    "host": "Anfitri\u00e3o"
                }
            }
        }
    }
}